tzlocal/unix.py

```diff
@@ -94,7 +94,7 @@ def _get_localzone_name(_root="/"):
     for filename in ("etc/sysconfig/clock", "etc/conf.d/clock"):
         tzpath = os.path.join(_root, filename)
         try:
-            with open(tzpath, "rt") as tzfile:
```

Comment:
Can we skip that UP015 rule? I like having the parameters there even if they are default.

Reply:
Updated.